includes/wc-product-functions.php

```diff
@@ -434,6 +434,9 @@ function wc_scheduled_sales() {
 
 			// Sync parent
 			if ( $parent ) {
+                                // Clear prices transient for variable products.
```

Comment:
Sorry, I noticed the spacing issue right after I submitted the pull request but I just did not want to crowd your code with a spacing fix.
I fixed the spacing and pushed it to the fork.